frontend/pages/_error.js

```diff
@@ -0,0 +1,65 @@
+import NextErrorComponent from 'next/error';
```

Comment:
This page was auto-generated, and I'm not sure we ever use it.

Reply:
More info on what this file is supposed to do: https://github.com/iFixit/react-commerce/compare/main...use-new-algolia-hooks

Reply:
Seeing a [warning](https://nextjs.org/docs/messages/custom-error-no-custom-404) in the console about this:

> [dev:frontend] [Next] Warning: You have added a custom /_error page without a custom /404 page. This prevents the 404 page from being auto statically optimized.
> [dev:frontend] [Next] See here for info: https://nextjs.org/docs/messages/custom-error-no-custom-404